app/views/users/index.html.erb

```diff
@@ -1 +1 @@
-<h1>Homepage where all users will be displayed</h1>
\ No newline at end of file
+<%= render partial: 'user', collection: @users %>
```

Comment:
- [ ] [REQUIRED] Please make sure that user can Click on each user to take to their show page